### PR TITLE
Chore: Improve the mobile view of the login page

### PR DESCRIPTION
**Pull request by amirajorloo**
**What this PR does / why we need it**:
The login page has some overflows in mobile devices (especially small ones), the logo and title are too big, and the footer links are untouchable because the login wrapper is above them.

**Which issue(s) this PR fixes**:
This PR will improve the mobile view of the login page. I tested it on iPhone 7, iPhone SE, and Galaxy S5:

| Before      | After |
| ----------- | ----------- |
| **iPhone 7**            |
|iPhone 7  |  iPhone 7     |
| **iPhone SE**            |
| iPhone SE |  iPhone SE |
| **Galaxy S5**            |
|  Galaxy S5 |  Galaxy S5  |
| **Alert**            |
| iPhone 7 Alert  |  iPhone 7 Alert |

- Automatically closes linked issue when the Pull Request is merged.

### CODE REVIEW

File: public/sass/components/_footer.scss [hunk 1]
```diff
@@ -59,3 +60,15 @@
     }
   }
 }
+
+@include media-breakpoint-down(sm) {
```
Comment (torkelo) [inline public/sass/components/_footer.scss:64]: we already have a @include media-breakpoint-up(sm) {  for this rule, so would be bettter to change the default style  and update existing media query (think mobile first, so default styles are for the mobile screens, and then media queries for larger screens )

Comment (theSinner) [reply public/sass/components/_footer.scss]: You're right. I just want to make fewer changes to the code. I changed it.

**Comment by kaydelaney** [comment]
Looks good! Since these changes were made with v7.5 in mind though, I should ask if you're happy with how they affect the v8 login screen, or if you think additional tweaks are needed. Nice job either way! @theSinner

**Review by kaydelaney — APPROVED** [review]

**Comment by theSinner** [comment]
> Looks good! Since these changes were made with v7.5 in mind though, I should ask if you're happy with how they affect the v8 login screen, or if you think additional tweaks are needed. Nice job either way! @theSinner

Yes, I did change it a bit to make it compatible with the V8 version, and I think it made it better.
@kaydelaney 

| Before      | After |
| ----------- | ----------- |
| **iPhone 7**            |
| iphone 7  | iphone 7 |
| **Galaxy S5**            |
| galaxy s5| galaxy s5 |
| **Alert**            |
|  error |  error |

**Pull request merged**